.github/workflows/performance-report.yml

```
@@ -0,0 +1,610 @@
+              // Calculate deltas
+              const avgDelta = current.avg - baseline.avg;
+              const avgPercent = ((avgDelta / baseline.avg) * 100).toFixed(1);
+              const maxDelta = current.max - baseline.max;
+              const maxPercent = current.max && baseline.max ? ((maxDelta / baseline.max) * 100).toFixed(1) : 0;
+              const minDelta = current.min - baseline.min;
+              const minPercent = current.min && baseline.min ? ((minDelta / baseline.min) * 100).toFixed(1) : 0;
+
+              // Replace latency table with comparison table
+              const latencyTableRegex = /(\| Metric \| Value \|\n\|--------|-------\|)([\s\S]*?)(\n\n)/;
+              let comparisonTable = `| Metric | Current | Baseline | Change |
+|--------|---------|----------|--------|
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected comment or line break

(syntax-check)